src/PayID/pay-id-client.ts

```diff
@@ -90,8 +90,11 @@ export default class PayIDClient {
       }
       // TODO(keefertaylor): make sure the header matches the request.
     } else if (data?.addresses) {
```

Comment:
Can we compress this?

```suggestion
    } else if (data?.addresses?.length === 1) {
```